policyengine_us/parameters/gov/states/hi/tax/income/additions.yaml

```diff
@@ -0,0 +1,17 @@
+description: Hawaii adds these sources to adjusted gross income.
+values:
+  2021-01-01:
+    # Line 8 state/fed wage diff
```

Comment:
I dont think this will pass the test with an empty list - need to delete and delete the adds in the variable

Reply:
> I dont think this will pass the test with an empty list - need to delete and delete the adds in the variable

copy that